deployment/terraform/assets/dashboard_data.json

```diff
@@ -1790,23 +1975,27 @@
     }
   ],
   "refresh": "10s",
-  "schemaVersion": 22,
-  "style": "dark",
+  "schemaVersion": 39,
   "tags": [
     "mattermost",
     "mattermost-perf"
   ],
   "templating": {
     "list": [
       {
-        "allValue": null,
         "current": {
-          "text": "",
+          "selected": true,
+          "text": [
+            "192.168.254.1:8067"
```

Comment:
Not sure if these should be set.

Reply:
Doesn't need to be. It was just part of the exported JSON file. I can remove them to keep it clean.